ProcessLib/HeatTransportBHE/BHE/BHE_CXC.h

```diff
@@ -0,0 +1,151 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Eigen>
+
+#include "BaseLib/Error.h"
+
+#include "BHECommon.h"
+#include "FlowAndTemperatureControl.h"
+#include "PipeConfigurationCXC.h"
+
+namespace ProcessLib
+{
+namespace HeatTransportBHE
+{
+namespace BHE
+{
+/**
+ * The BHE_CXC class is the realization of Coaxial pipe with Centred type of the
+ * Borehole Heate Exchanger. In this class, the pipe heat capacity,
+ * pipe heat conductiion, pie advection vectors are intialized according to the
+ * geometry of CXC type of BHE. For CXC type of BHE, 3 primary unknowns are
+ * assigned on the 1D BHE elements. They are the temperature in inflow pipe
+ * T_in, temperature in outflow pipe T_out, temperature of the grout zones
+ * sorrounding the inflow pipe T_g. These primary varaibles are solved
+ * according to heat convection and conduction equations on the pipes and also
+ * in the grout zones. The interaction of the CXC type of BHE and the
+ * sorrounding soil is regulated through the thermal resistance values, which
+ * are calculated specifically during the initialization of the class.
```

Comment:
The documentation contains the same spelling mistakes as the documentation for class `BHE_CXA`.

Reply:
Corrected.